frontend/src/shared/HelloWorldPage/hello-world-page.scss

```diff
@@ -0,0 +1,21 @@
+.hello-world-page {
+    width: 100%;
+    height: 100%;
+
+    text-align: center;
+
+    background-image: url("image.png");
+    background-repeat: no-repeat;
+}
+
+.hello-world-page__content {
+    display: inline;
+
+    margin-top: 10%;
+
+    text-align: center;
+
+    background-color: white;
+
+    font-size: 200%;
```

Comment:
cleanup empty lines

Reply:
I just can't clean up those lines. Stylelint doesn't let me to.
With an error:
![image](https://user-images.githubusercontent.com/25225968/36582106-26b18efc-1881-11e8-8702-c63229e19b0c.png)
at each line, don't know how to handle this one.